src/SourceBuild/tarball/patches/installer/0001-non-portable-build.patch

```diff
@@ -0,0 +1,156 @@
+From 9b884877f84a052f7d915db791acef4312cb8bab Mon Sep 17 00:00:00 2001
+From: Tom Deseyn <tom.deseyn@gmail.com>
+Date: Mon, 19 Sep 2022 07:45:00 +0200
+Subject: [PATCH] Rename MicrosoftAspNetCoreAppRuntimePackageVersion
+
+---
+ eng/Versions.props                                   |  4 ++--
+ src/redist/targets/GenerateBundledVersions.targets   |  4 ++--
+ src/redist/targets/GenerateDebs.targets              |  2 +-
+ src/redist/targets/GenerateLayout.targets            | 12 ++++++------
+ src/redist/targets/GenerateMSIs.targets              |  2 +-
+ src/redist/targets/GeneratePKG.targets               |  2 +-
+ .../LinuxNativeInstallerDependencyVersions.targets   |  2 +-
+ 7 files changed, 14 insertions(+), 14 deletions(-)
+
+diff --git a/eng/Versions.props b/eng/Versions.props
+index 87b644061..7866f90e0 100644
+--- a/eng/Versions.props
++++ b/eng/Versions.props
+@@ -96,7 +96,7 @@
+     <!-- version above and create aliases without the winx64 here for clarity elsewhere. -->
+     <MicrosoftNETCoreAppHostPackageVersion>$(MicrosoftNETCoreAppHostwinx64PackageVersion)</MicrosoftNETCoreAppHostPackageVersion>
+     <MicrosoftNETCoreAppRuntimePackageVersion>$(MicrosoftNETCoreAppRuntimewinx64PackageVersion)</MicrosoftNETCoreAppRuntimePackageVersion>
+-    <MicrosoftAspNetCoreAppRuntimePackageVersion>$(MicrosoftAspNetCoreAppRuntimewinx64PackageVersion)</MicrosoftAspNetCoreAppRuntimePackageVersion>
++    <MicrosoftAspNetCoreAppRuntimePackageVersionBuilt>$(MicrosoftAspNetCoreAppRuntimewinx64PackageVersion)</MicrosoftAspNetCoreAppRuntimePackageVersionBuilt>
```

Comment:
I've added this to deal with https://github.com/dotnet/installer/issues/14492. This renames the property so source-build doesn't override it. There are better ways to fix this. I noticed the issue after I wrote this patch.